test/resources/stablex_utils.js

```diff
@@ -0,0 +1,91 @@
+const MockContract = artifacts.require("MockContract")
+const StablecoinConverter = artifacts.require("StablecoinConverter")
+
+const {
+  waitForNSeconds,
+  sendTxAndGetReturnValue,
+} = require("../utilities")
+
+/**
+ * @typedef Deposit
+ * @type {object}
+ * @property {BN} amount The deposit amount
+ * @property {number} token The deposited token
+ * @property {number} user The user making the deposit
+ */
+
+/**
+ * Makes deposit transactions from a list of Deposit Objects
+ * @param {number} numTokens
+ * @param {number} maxTokens
+ * @param {number} feeDenominator
+ * @returns {}
+ */
+const setupGenericStableX = async function (numTokens = 2, maxTokens = 2 ** 16 - 1, feeDenominator = 1000) {
+  const feeToken = await MockContract.new()
+  const instance = await StablecoinConverter.new(maxTokens, feeDenominator, feeToken.address)
+  await feeToken.givenAnyReturnBool(true)
+  const tokens = [feeToken]
+  for (let i = 0; i < numTokens - 1; i++) {
+    const token = await MockContract.new()
+    await instance.addToken(token.address)
+    await token.givenAnyReturnBool(true)
+    tokens.push(token)
+  }
+  return instance
+}
+
+
+/**
+ * Makes deposit transactions from a list of Deposit Objects
+ * @param {contract} - StablecoinConverter Smart Contract
+ * @param {accounts} - an array of (unlocked) ethereum account addresses
+ * @param {Deposit[]} - an array of Deposit Objects
+ */
```

Comment:
I think here the types aren't what I expected. And the parameter names are missing. Maybe something like:
```
 * @param {object} contract StablecoinConverter smart contract
 * @param {string[]} accounts An array of ...
 * @param {Deposit[]} depolitList An array of ...
```